modules/apiserversource-yaml.adoc

```diff
@@ -115,15 +112,15 @@ spec:
       containers:
         - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
 ----
-+
-After you have created the `service.yaml` file, apply it:
+
+. Apply the `Service` YAML file:
 +
 [source,terminal]
 ----
-$ oc apply -f service.yaml
+$ oc apply -f <filename>
 ----
 
-. To create a trigger from the `default` broker that filters events to the service created in the previous step, create a file named `trigger.yaml` and copy the following sample code into it:
+. Create a `Trigger` object as a YAML file, that filters events from the `default` broker to the service created in the previous step:
```

Comment:
I think you can drop the comma after file now that the sentence is restructured and no longer has an introductory clause